lib/compat/wordpress-5.9/render-svg-filters.php

```diff
@@ -5,22 +5,44 @@
  * @package gutenberg
  */
 
-/**
- * Render the SVG filters supplied by theme.json.
- *
- * Note that this doesn't render the per-block user-defined
- * filters which are handled by duotone.php, but it should
- * be rendered in the same location as those to satisfy
- * Safari's rendering quirks.
+/*
+ * If wp_global_styles_render_svg_filters is defined, it means the plugin
+ * is running on WordPress 5.9.1, so don't need to render the global styles
+ * SVG filters as it was already done by WordPress core.
  */
-function gutenberg_experimental_global_styles_render_svg_filters() {
-	$filters = wp_get_global_styles_svg_filters();
-	if ( ! empty( $filters ) ) {
-		echo $filters;
+if ( ! function_exists( 'wp_global_styles_render_svg_filters' ) ) {
+	/**
+	 * Render the SVG filters supplied by theme.json.
+	 *
+	 * Note that this doesn't render the per-block user-defined
+	 * filters which are handled by wp_render_duotone_support,
+	 * but it should be rendered before the filtered content
+	 * in the body to satisfy Safari's rendering quirks.
+	 */
+	function gutenberg_experimental_global_styles_render_svg_filters() {
```

Comment:
If we provide this function only as a backup for when the core doesn't, we tend to use the same name: `wp_global_styles_render_svg_filters`.